src/cart/crt_rpc.c

```diff
@@ -162,59 +151,48 @@ crt_proc_struct_crt_grp_cache(crt_proc_t proc, crt_proc_op_t proc_op,
  */
 CRT_RPC_DEFINE(crt_ctl_get_uri_cache, CRT_ISEQ_CTL, CRT_OSEQ_CTL_GET_URI_CACHE)
-CRT_RPC_DEFINE(crt_ctl_ep_ls,         CRT_ISEQ_CTL, CRT_OSEQ_CTL_EP_LS)
-CRT_RPC_DEFINE(crt_ctl_get_host,      CRT_ISEQ_CTL, CRT_OSEQ_CTL_GET_HOST)
-CRT_RPC_DEFINE(crt_ctl_get_pid,       CRT_ISEQ_CTL, CRT_OSEQ_CTL_GET_PID)
+CRT_RPC_DEFINE(crt_ctl_ep_ls, CRT_ISEQ_CTL, CRT_OSEQ_CTL_EP_LS)
+CRT_RPC_DEFINE(crt_ctl_get_host, CRT_ISEQ_CTL, CRT_OSEQ_CTL_GET_HOST)
+CRT_RPC_DEFINE(crt_ctl_get_pid, CRT_ISEQ_CTL, CRT_OSEQ_CTL_GET_PID)
 
 CRT_RPC_DEFINE(crt_proto_query, CRT_ISEQ_PROTO_QUERY, CRT_OSEQ_PROTO_QUERY)
 
-CRT_RPC_DEFINE(crt_ctl_fi_attr_set, CRT_ISEQ_CTL_FI_ATTR_SET,
-	       CRT_OSEQ_CTL_FI_ATTR_SET)
-CRT_RPC_DEFINE(crt_ctl_fi_toggle, CRT_ISEQ_CTL_FI_TOGGLE,
-	       CRT_OSEQ_CTL_FI_TOGGLE)
+CRT_RPC_DEFINE(crt_ctl_fi_attr_set, CRT_ISEQ_CTL_FI_ATTR_SET, CRT_OSEQ_CTL_FI_ATTR_SET)
+CRT_RPC_DEFINE(crt_ctl_fi_toggle, CRT_ISEQ_CTL_FI_TOGGLE, CRT_OSEQ_CTL_FI_TOGGLE)
 
 CRT_RPC_DEFINE(crt_ctl_log_set, CRT_ISEQ_CTL_LOG_SET, CRT_OSEQ_CTL_LOG_SET)
-CRT_RPC_DEFINE(crt_ctl_log_add_msg, CRT_ISEQ_CTL_LOG_ADD_MSG,
-	       CRT_OSEQ_CTL_LOG_ADD_MSG)
+CRT_RPC_DEFINE(crt_ctl_log_add_msg, CRT_ISEQ_CTL_LOG_ADD_MSG, CRT_OSEQ_CTL_LOG_ADD_MSG)
 
 /* Define for crt_internal_rpcs[] array population below.
  * See CRT_INTERNAL_RPCS_LIST macro definition
  */
-#define X(a, b, c, d, e)	\
-{				\
-	.prf_flags = b,		\
-	.prf_req_fmt = c,	\
-	.prf_hdlr = d,		\
-	.prf_co_ops = e,	\
-},
-
-static struct crt_proto_rpc_format crt_internal_rpcs[] = {
-	CRT_INTERNAL_RPCS_LIST
-};
+#define X(a, b, c, d, e)                                                                           \
+	{                                                                                          \
+	    .prf_flags   = b,                                                                      \
+	    .prf_req_fmt = c,                                                                      \
+	    .prf_hdlr    = d,                                                                      \
+	    .prf_co_ops  = e,                                                                      \
+	},
 
-static struct crt_proto_rpc_format crt_fi_rpcs[] = {
-	CRT_FI_RPCS_LIST
-};
+static struct crt_proto_rpc_format crt_internal_rpcs[] = {CRT_INTERNAL_RPCS_LIST};
 
-static struct crt_proto_rpc_format crt_st_rpcs[] = {
-	CRT_ST_RPCS_LIST
-};
+static struct crt_proto_rpc_format crt_fi_rpcs[] = {CRT_FI_RPCS_LIST};
 
-static struct crt_proto_rpc_format crt_ctl_rpcs[] = {
-	CRT_CTL_RPCS_LIST
-};
+static struct crt_proto_rpc_format crt_st_rpcs[] = {CRT_ST_RPCS_LIST};
 
-static struct crt_proto_rpc_format crt_iv_rpcs[] = {
-	CRT_IV_RPCS_LIST
-};
+static struct crt_proto_rpc_format crt_ctl_rpcs[] = {CRT_CTL_RPCS_LIST};
+
+static struct crt_proto_rpc_format crt_iv_rpcs[] = {CRT_IV_RPCS_LIST};
 
 #undef X
 
-#define X(a, b, c, d, e) case a: return #a;
+#define X(a, b, c, d, e)                                                                           \
```

Comment:
(style)  Macros with multiple statements should be enclosed in a do - while loop
(style)  Macros with flow control statements should be avoided
(style)  macros should not use a trailing semicolon